packages/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTHost.mm

```diff
@@ -320,24 +279,19 @@ - (void)didReceiveReloadCommand
     [self _setBundleURL:_bundleURLProvider()];
   }
 
-  // Ensure all attached surfaces are restarted after reload
-  {
-    std::lock_guard<std::mutex> guard{_surfaceStartBufferMutex};
-    _surfaceStartBuffer = [NSMutableArray arrayWithArray:[self _getAttachedSurfaces]];
-  }
-
   _instance = [[RCTInstance alloc] initWithDelegate:self
                                    jsRuntimeFactory:[self _provideJSEngine]
                                       bundleManager:_bundleManager
                          turboModuleManagerDelegate:_turboModuleManagerDelegate
-                                onInitialBundleLoad:_onInitialBundleLoad
+                                onInitialBundleLoad:nil
                                      moduleRegistry:_moduleRegistry
                               parentInspectorTarget:_inspectorTarget.get()
                                       launchOptions:_launchOptions];
   [_hostDelegate hostDidStart:self];
 
   for (RCTFabricSurface *surface in [self _getAttachedSurfaces]) {
     [surface resetWithSurfacePresenter:self.surfacePresenter];
+    [_instance callFunctionOnBufferedRumtimeExecutor:[surface](facebook::jsi::Runtime &_) { [surface start]; }];
```

Comment:
Just a small typo "rumtime"

Reply:
Thanks, good catch, I would fix it in my next PR. :)